src/Widgets/ToolbarButton.vala

```diff
@@ -110,4 +121,8 @@ public class ENotes.ToolbarButton : Gtk.Button {
             }
         });
     }
+
+    private bool already_applied (string text, string first_half, string second_half) {
```

Comment:
You should make this function static since it doesn't modify things outside of the scope of your function :) 

_I know i didn't do this on the rest of my code.... but Notes-Up is pretty old :sweat_smile"_